src/plugins/d3/renderer/hooks/useChartOptions/y-axis.ts

```diff
@@ -50,10 +51,27 @@ const getAxisLabelMaxWidth = (args: {axis: PreparedAxis; series: ChartKitWidgetS
 
 function getAxisMin(axis?: ChartKitWidgetAxis, series?: ChartKitWidgetSeries[]) {
     const min = axis?.min;
-    const seriesWithVolume = ['bar-x', 'area'];
+    const seriesWithVolume = ['bar-x', 'area', 'waterfall'];
 
     if (typeof min === 'undefined' && series?.some((s) => seriesWithVolume.includes(s.type))) {
-        return 0;
+        return series.reduce((minValue, s) => {
+            switch (s.type) {
+                case 'waterfall': {
+                    const minSubTotal = s.data.reduce(
+                        (res, d) =>
+                            Math.min(
```

Comment:
I'm not sure if this makes sense, since there will be different calculations for the Y-axis value for different series. As a result, only reduce will remain in the helper